### DISCUSSION TITLE
rec v4 官网识别模型，英文是用哪个数据集训练？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#13945] [Q&A]
rec v4 官网识别模型，英文是用哪个数据集训练？****

**Comment by UserWangZz** [comment]
公开数据集+内部数据集

**Comment by nissansz** [comment]
哪个公开数据集好？









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: WangZ  ***@***.***>
 

 
 发送时间: 2024-10-09 12:04:48
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] rec v4 官网识别模型，英文是用哪个数据集训练？ (Discussion #13945)
 





 公开数据集+内部数据集

**Reply by UserWangZz** [reply]
这个和你场景需求有关吧

**Comment by nissansz** [comment]
想要训练通用模型。打印文件，各种字体，ppt扫描，字幕等。
现在训练到0.62左右准确率，效果还可以。

有的字体效果差。








--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: WangZ  ***@***.***>
 

 
 发送时间: 2024-10-15 11:17:53
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] rec v4 官网识别模型，英文是用哪个数据集训练？ (Discussion #13945)
 





 这个和你场景需求有关吧
